bridge_adaptivity/module/policies/policy_trials_count.py

```diff
@@ -4,6 +4,9 @@
 class TrialsCountGradingPolicy(BaseGradingPolicy):
     public_name = 'Trials count'
 
+    summary_text = """ Overall score is the number of attempts made divided by the threshold Q, or 1 if the number of
+    attempts made is greater than Q."""
+
```

Comment:
Have you missed to add detail text, or it is absent in the wiki?

Reply:
added and reviewed by @kunanit